backend/dal/Helpers/Extensions/PropertyExtensions.cs

```diff
@@ -7,7 +7,7 @@
 using System.Linq;
 using System.Security.Claims;
 using Entity = Pims.Dal.Entities;
-using Pims.Dal.Helpers.Constants;
+using Pims.Dal.Entities;
```

Comment:
not sure if this matters, but you have "using Pims.Dal.Entities" and then two lines above it "using Entity = Pims.Dal.Entities"